routes/routes.js

```diff
@@ -25,10 +25,13 @@ module.exports = function(app, passport) {
     })
   })
 
-  router.post('/create', function(req,res, next){
+  router.post('/:gameid/:userid', isLoggedIn, function(red,res,next){
```

Comment:
Is this for joining or creating a game? Says you removed `/create` so I assume creating? If so, I would suggest leaving it as /create. I can take care of routes though as this has been my priority right now